src/callout.tsx

```diff
@@ -4,41 +4,18 @@ import Inline from './inline';
 import Box from './box';
 import {useTheme} from './hooks';
 import {ThemeVariant, useIsInverseVariant} from './theme-variant-context';
-import {createUseStyles} from './jss';
 import {Text2, Text3} from './text';
 import IconCloseRegular from './generated/mistica-icons/icon-close-regular';
 import IconButton from './icon-button';
 import classNames from 'classnames';
 import {ButtonLink, ButtonPrimary, ButtonSecondary} from './button';
 import ButtonGroup from './button-group';
+import * as styles from './callout.css';
+import {sprinkles} from './sprinkles.css';
+import {vars} from './skins/skin-contract.css';
 
 import type {RendersNullableElement} from './utils/types';
 
-const useStyles = createUseStyles(({colors}) => ({
-    container: {
-        background: colors.backgroundAlternative,
-        borderRadius: 8,
-        padding: 16,
-        overflow: 'hidden',
-        display: 'flex',
-        minHeight: 56,
-        '& > *': {
-            flexShrink: 0,
-        },
```

Comment:
not needed?

Reply:
dont think so, only two direct children, `.content`, that already has a explicit `flex: 1`, and the icon, that has a fixed size... so I dont know what this think were doing... I tried it in storybook and all seems ok